### ISSUE TITLE
Issues related to plot_histograms function

### ISSUE THREAD

**Issue by AraiYuno** [UBC-MDS/snapedautility#2]
Here is a git issue to keep track of all the issues and improvements for `plot_histograms` function.

**Comment by AraiYuno** [comment]
@harryyikhchan 
![image](https://user-images.githubusercontent.com/28745975/151297188-e6d9439e-4e3e-4db9-9a54-93f304a10573.png)
Based on your error messages, I diagnosed there is an error that I did not realize during initial testing.

![image](https://user-images.githubusercontent.com/28745975/151296925-c0884388-38fb-4bca-bc43-883c78fe7507.png)
![image](https://user-images.githubusercontent.com/28745975/151297287-8bfeed7c-b9f9-4574-8095-f1e0bde684cc.png)
I have made the fixes and confirmed that all the unit tests still pass. I have also confirmed that `plot_histograms` function runs fine on a jupyter notebook.
